src/main/features/response/form/models/timelineRow.ts

```diff
@@ -0,0 +1,50 @@
+import { IsDefined, ValidateIf } from 'class-validator'
+
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+
+export class ValidationErrors {
+  static readonly DATE_REQUIRED: string = 'Enter date'
+  static readonly DESCRIPTION_REQUIRED: string = 'Enter description'
+}
+
+export class TimelineRow {
+
+  @ValidateIf(o => o.description !== undefined)
```

Comment:
this looks wrong, the only validation on this is maximum 20 characters and only if its not blank

Reply:
done